internal/service/servicediscovery/service.go

```diff
@@ -123,9 +123,10 @@ func resourceService() *schema.Resource {
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"failure_threshold": {
-							Type:     schema.TypeInt,
-							Optional: true,
-							ForceNew: true,
+							Type:       schema.TypeInt,
+							Optional:   true,
+							ForceNew:   true,
+							Deprecated: "failure_threshold is deprecated and no longer supported by AWS. The attribute will be removed in a future major version.",
```

Comment:
We should maybe mention here that AWS always sets this value to `1` since it'll display any time the argument is set. 

That way if the user misses the registry documentation or CHANGELOG they'll still have appropriate expectations for the behavior.